README.md

```diff
@@ -92,13 +92,14 @@ Search is performed by Alpha-Beta-Pruning at a deepness of 4.
 The heuristic evaluation function is based on various considerations:
 
 - Total pawns:
-  - The more pawns one has compared to the number of pawns the opponent has, the better.
+  - The more pawns one has compared to the number of pawns the opponent has, the better. _(implemented)_
```

Comment:
remove not implemented information

Reply:
included but separated and explained why not implemented.